ldapauth.go

```diff
@@ -413,11 +413,9 @@ func Connect(addr string, port uint16, useTLS bool, startTLS bool, skipVerify bo
 		// we assume that error is due to missing port.
 		host = u.Host
 	}
-	LoggerDEBUG.Printf("Host: %s ", host)
 
-	address := net.JoinHostPort(host, strconv.FormatUint(uint64(port), 10))
-
-	LoggerDEBUG.Printf("Connect Address: %s ", address)
+	address := u.Scheme + "://" + net.JoinHostPort(host, strconv.FormatUint(uint64(port), 10))
+	LoggerDEBUG.Printf("Connect Address: '%s'", address)
 
 	if useTLS {
```

Comment:
Do you think we could just decommission the UseTLS option ? Technically if the scheme is `ldaps` or the `StartTLS` option is set to true, it could be inferred.

As it stands, we can already perform a successful TLS connection even with UseTLS set to false as long as StartTLS and custom CAs are not needed.

Reply:
> Do you think we could just decommission the UseTLS option ? Technically if the scheme is `ldaps` or the `StartTLS` option is set to true, it could be inferred.

I agree, this will also prevent errors when setting `UseTLS` and `StartTLS` both to `true` when using a `ldaps` server. In this case, a simple `ldap.DialURL(address)` will be made without TLS config.

https://github.com/wiltonsr/ldapAuth/blob/960ebf858f21bc835967b800f1f8ada426e0bfab/ldapauth.go#L426-L427